<?xml version="1.0"?>
<launch>

  <group ns="mcr_perception">

    <node pkg="mcr_object_list_merger" type="object_list_merger" name="object_list_merger" output="screen">
      <param name="distance_threshold" type="double" value="0.02" />
      <param name="use_pose_average" type="bool" value="false" />
      <remap from="~input_object_list" to="/mcr_perception/object_detector/object_list" />
      <remap from="~knowledge_base_query_topic" to="/mir_knowledge_base_analyzer/knowledge_base_queries/query" />
      <remap from="~knowledge_base_objects_list" to="/mir_knowledge_base_analyzer/knowledge_base_queries/objects_at_location" />
    </node>

  </group>

</launch>
